docs/project/dogfooding.md

```diff
@@ -6,6 +6,23 @@ This document provides the steps necessary to consume a nightly build of
 Please note that these steps are likely to change as we're simplifying
 this experience. Make sure to consult this document often.
 
+## Obtaining nightly builds of NuGet packages
+
+If you are only looking to get fixes for an individual NuGet package, and don't need a preview version of the entire runtime, you can add the following package feed to `NuGet.config`:
+
+**MSFT TODO:** Someone at Microsoft should confirm if any other feeds might be needed here
+
+```xml
+<packageSources>
+    <add key="dotnet-core" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json" />
+    ...
+</packageSources>    
+```
+
+(Documentation for configuring feeds is [here](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior).)
```

Comment:
This link refers to the documentation for the old nuget.exe that is not really a thing with .NET Core.

It may be better to refer to https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-add-source

Reply:
Makes sense to me.  I was mainly just copying to be consistent with what you had further down in the document.